build/gradle/updateSite.gradle

```diff
@@ -159,7 +159,7 @@ task buildUpdateSite(dependsOn: [ createUpdateSiteFeature, generatePomFiles ]) {
 			]
 		}
 	    pg.generateParentPomFile(parentPomConfig)
-		
+		System.setProperty( MavenCli.MULTIMODULE_PROJECT_DIRECTORY, projectRoot);
```

Comment:
Indentation seems off

Reply:
No idea why is it like this here, in my workspace this is exactly below `pg.` 

```
pg.generateParentPomFile(parentPomConfig) -> Line:column (161:9)
System.setProperty()                      -> Line:column (162:9)
```

Reply:
I tried to add a blank like but still it is the same here in GitHub. However, in GitHub desktop it looks properly indented. 
![Screenshot 2022-11-21 at 15 09 17](https://user-images.githubusercontent.com/28185404/203075584-6162e94b-3bed-484d-b6f9-d6cb9194206e.png)

Reply:
Probably inconsistent use of indentation in the file then. Then leave it as it is I would say.